data_engineering/pyspark_tasks/task_1.py

```diff
@@ -0,0 +1,47 @@
+"""This module saves top 100 films according to IMDB ratings(all time,60th,last 10 years)"""
+from pyspark import SparkConf
+from pyspark.sql import SparkSession
+
+
+def get_best_alltime(basics_df, ratings_df):
+    return basics_df.join(ratings_df, basics_df['tconst'] == ratings_df['tconst'], 'inner') \
+        .select(basics_df['tconst'], 'primaryTitle', 'numVotes', 'averageRating', 'startYear') \
+        .where('numVotes>=100000 and titleType="movie"') \
+        .orderBy('averageRating', ascending=False).limit(100)
+
+
+def get_best_60th(basics_df,ratings_df):
+    return basics_df.join(ratings_df, basics_df['tconst'] == ratings_df['tconst'], 'inner') \
+        .select(basics_df['tconst'], 'primaryTitle', 'numVotes', 'averageRating', 'startYear') \
+        .where('numVotes>=100000 and titleType="movie" and startYear between 1960 and 1969') \
+        .orderBy('averageRating', ascending=False).limit(100)
+
+
+def get_best_decade(basics_df,ratings_df):
+    return basics_df.join(ratings_df, basics_df['tconst'] == ratings_df['tconst'], 'inner') \
+        .select(basics_df['tconst'], 'primaryTitle', 'numVotes', 'averageRating', 'startYear') \
+        .where('numVotes>=100000 and titleType="movie" and YEAR(current_date())-startYear<10') \
+        .orderBy('averageRating', ascending=False).limit(100)
+
+
+spark_session = (SparkSession.builder
+                 .master('local')
+                 .appName('task_1')
+                 .config(conf=SparkConf())
+                 .getOrCreate())
+
+
+ratings_df_main = spark_session.read.csv(path='input/title.ratings.tsv.gz', sep='\t', header=True)
```

Comment:
This code will be executed when importing from this module even though you might not need it, please place it somewhere in the function